packages/casting/src/defaults.js

```diff
@@ -39,7 +39,7 @@ export const randomBackoff = (range, cap = range) => {
 /**
  * @param {number} [attempt]
  * @param {number} [base]
- * @param {number} [cap=DEFAULT_MAX_BACKOFF]
+ * @param {number} [cap]
```

Comment:
help me understand how this change relates to the scope of this commit and this PR?

If it's a drive-by, a separate commit would have been nice. not critical.